### ISSUE TITLE
CPU increasing to 100% on one core with only opus encoding and bypass

### ISSUE THREAD

**Issue by nemzo89** [AirenSoft/OvenMediaEngine#97]
Yesterday I tried to build the new master branch to do a project with bypass and only opus encoding enabled. Since 0.10 did not build the SSL module, I cloned the master git branch and built it with the URL fix. Even though there is no other encoding enabled than opus, one thread quickly jumps up to 80% utilization on one core, and increases to 100% within some seconds which results in network packet drops in OBS as the source is not able to deliver to the server. All the other cores apart from this one on 100% have a quite low load, and the server easily did both H264 and VP8 encodes simultaneously in 0.9.1. Both versions also have TLS enabled.

Do you have any idea what could cause this CPU load?

**Comment by getroot** [comment]
Have you ever compiled with make release? We recently patched a bug in the release where the cpu load could go up to the dev branch (0d34747e9ab8a94bfcc66a399be2dfc6102cc621). Could you please check if this solves your problem?

**Comment by nemzo89** [comment]
I did build the 0.9.1 releases from source, but this is the first time I did a github fetch from ovenmediaengine. I did fetch the master and not dev branch, though. If you are not experiencing the same, I guess the error is on my side. I will do some testing with the latest releases and change you mentioned.

My process was:
Clean Ubuntu 18.04 on a cloud server

Commands:
git clone https://github.com/AirenSoft/OvenMediaEngine
cd OvenMediaEngine/
misc/prerequisites.sh
cd src
make release
sudo make install
systemctl start ovenmediaengine
systemctl enable ovenmediaengine.service 

During the first run I got an error with something like “Can’t exec aclocal” and “No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 326. ffplay". I figured that the default install of Ubuntu 18.04 on my cloud provider did have autoconf, but automake was not preinstalled, so after installing automake through apt the install process went smoothly.
